src/server/logMailer.js

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const nodemailer = require('nodemailer');
+const mg = require('nodemailer-mailgun-transport');
+const config = require('./config');
+const log = require('./log').log;
+
+/**
+ * Send an e-mail representing an error message.
+ * @param {string} level The level, as a string
+ * @param {string} message The string to e-mail
+ */
+function logMailer(level, message) {
+
+	var mailOptions = {
+		from: config.mailer.from,
+		to: config.mailer.to,
+		subject: `[OED ${config.mailer.org}] Open Energy Dashboard ${level}`,
+		text: message
+	};
+
+	let transporter;
+	if (config.mailer.method === 'none') {
+		return;
+	} else if (config.mailer.method === 'mailgun') {
+		transporter = nodemailer.createTransport(mg({
```

Comment:
I think it would be nicer if you moved the declarations of the transporters outside the function call.